core/services/pg/q.go

```diff
@@ -119,11 +119,15 @@ func NewQ(db *sqlx.DB, logger logger.Logger, config LogConfig, qopts ...QOpt) (q
 		q.Queryer = db
 	}
 	q.db = db
-	q.logger = logger
+	q.logger = logger.Helper(2)
```

Comment:
Skip two levels to point to the callers with the actual SQL instead of the inner helper methods. For example:
```
orm:67	err = q.Get(head, `SELECT * FROM heads WHERE evm_chain_id = $1 ORDER BY number DESC, created_at DESC, id DESC LIMIT 1`, orm.chainID)
```
instead of
```
pg:283	q.logger.Warnf("SLOW SQL QUERY: %d ms", elapsed.Milliseconds())
```